### PR TITLE
Reproducible results

### PR DESCRIPTION
**Pull request by cnpgs**
Implementation for more consistent results via setting of a seed.

### CODE REVIEW

File: actableai/tasks/base.py [hunk 1]
```diff
@@ -249,6 +254,15 @@ def _func(task_object, *args, **kwargs):
                 import logging
                 import tensorflow as tf
 
+                if task_object.seed is not None:
```
Comment (trunghlt) [inline actableai/tasks/base.py:257]: Hmnn doesn't it only apply to functions that use tensorflow?

Comment (cnpgs) [reply actableai/tasks/base.py:257]: As mentioned in the code comments, `tf.keras.utils.set_random_seed(task_object.seed)` is equivalent to calling `random.seed()`, `np.random.seed()`, and `tf.random.set_seed()` - so it sets all random seeds for the program (Python, NumPy, and TensorFlow), not just the TensorFlow seed.

More information can also be viewed at https://www.tensorflow.org/api_docs/python/tf/keras/utils/set_random_seed

**Review by trunghlt — APPROVED** [review]

**Review by trunghlt — APPROVED** [review]
